great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -331,8 +334,13 @@ def _init_factories(self) -> None:
                 context=self,
             )
 
-        # TODO: Update to follow existing pattern once new ValidationConfigStore is implemented
         self._validations: ValidationFactory | None = None
+        if validation_config_store := self.stores.get(
```

Comment:
Is there no default store, eg an ephemeral store?

Reply:
Currently no - we allow optional configuration but won't raise an error until it is used in CRUD.

We should change this but prioritizing feature development first. I am ensuring that all new contexts create this store though

Reply:
```suggestion
        validation_config_store = self.validation_config_store # or something like that
```